src/security/tests/srv_acl_tests.c

```diff
@@ -31,13 +31,15 @@ char *ds_sec_server_socket_path = "/fake/socket/path";
  */
 #define TEST_USER	"myuser@"
 #define TEST_GROUP	"mygroup@"
+#define TEST_HOST	"testhost"
```

Comment:
Should add some unit tests for your new ds_sec_get_origin function

Reply:
You're right. I had added tests when it was part of the other function but I forgot to add ones for the new function.

Reply:
Added a bunch of tests.